spec/overcommit/hook_context/pre_push_spec.rb

```diff
@@ -121,7 +169,7 @@
           `git add . 2>&1 > #{File::NULL}`
           `git commit -m "Update Branch 2" 2>&1 > #{File::NULL}`
 
-          should include(*%w[added-1 update-me added-2].map { |file| File.expand_path(file) })
+          should == %w[added-1 added-2 update-me].map { |file| File.expand_path(file) }
```

Comment:
One thing worth noting is that using `==` here is slightly more brittle, as it prevents us from changing our implementation (which could return the file names in a different order) without breaking tests.

At the end of the day, our goal is to make sure we're returning the correct _set_ of files, not files in a specified order, which is why we used `include`. However, perhaps you changed this because we weren't checking for unintended files to be included. In that case, you could use a combination of `include` along with verifying the count to make sure no additional files are in the set.

This is not a big deal, so I'm fine merging this as-is, but it's important we're not overly-specific in tests.

Reply:
I’ll take a note for future reference. Thanks.